AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/RequestController.cs

```diff
@@ -98,21 +102,28 @@ public RequestController(IMediator mediator)
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> Edit(EditRequestViewModel model)
         {
-            var campaignEvent = await _mediator.SendAsync(new EventSummaryQuery { EventId = model.EventId });
+            if (!ModelState.IsValid)
+            {
+                return View("Edit", model);
+            }
 
-            if (campaignEvent == null)
+            var @event = await _mediator.SendAsync(new EventSummaryQuery { EventId = model.EventId });
+            if (@event == null)
             {
                 return BadRequest("Invalid event id");
             }
 
-            if (!User.IsOrganizationAdmin(campaignEvent.OrganizationId))
+            if (!User.IsOrganizationAdmin(@event.OrganizationId))
             {
                 return Unauthorized();
             }
 
-            if (!ModelState.IsValid)
+            var validatePhoneNumberResult = await _mediator.SendAsync(new ValidatePhoneNumberRequest { PhoneNumber = model.Phone, ValidateType = true });
+            if (!validatePhoneNumberResult.IsValid)
             {
-                return View("Edit", model);
+                var phoneAttribute = model.GetType().GetProperties().Single(p => p.Name == nameof(model.Phone)).GetCustomAttributes(false).Cast<Attribute>().Single(a => a is PhoneAttribute) as PhoneAttribute;
```

Comment:
I'm not sure we need the reflection stuff here to get a message. We can pass whatever we want into the AddModelError as a string so maybe we just hard code a const to represent the "Invalid phone number" message?